index.html

```diff
@@ -11,33 +11,3186 @@
+        },
+        {
+          name: "Shannon Appelcline"
+        },
+        {
+          name: "Joachim Lohkamp"
+        },
+        {
+          name: "Drummond Reed",
+          url: "https://www.linkedin.com/in/drummondreed/",
+          company: "Evernym",
+          companyURL: "https://www.evernym.com/",
+          w3cid: 3096
+        },
+        {
+          name: "Markus Sabadello",
+          url: "https://www.linkedin.com/in/markus-sabadello-353a0821",
+          company: "Danube Tech",
+          companyURL: "https://danubetech.com/",
+          w3cid: 46729
+        },
+        {
+          name: "Oliver Terbu"
+        },
+        {
+          name: "Amy Guy"
+        }
+        ],
+
+        maxTocLevel: 3,
+        localBiblio: {
+          "DID-CORE": {
+            title: "DID Core Specification",
+            href: "https://www.w3.org/TR/did-core",
+            authors: [
+              "Manu Sporny",
+              "Markus Sabadello",
+              "Drummond Reed"
+            ],
+            status: "WD",
+            publisher: "Decentralized Identifier Working Group"
+          },
+          "DID-SPEC-REGISTRIES": {
+            title: "DID Specification Registries",
+            href: "https://w3c.github.io/did-spec-registries/",
+            authors: [
+              "Orie Steele",
+              "Manu Sporny"
+            ],
+            status: "ED",
+            publisher: "Decentralized Identifier Working Group"
+          }
+        }
       };
     </script>
   </head>
   <body>
     <section id="abstract">
       <p>
-        This is just a placeholder for now.
+The communities behind Decentralized Identifiers (DIDs) bring together a
+diverse group of contributors who have decidedly different notions of exactly
+what "decentralization" means.
       </p>
-    </section>
-    <section id="sotd">
       <p>
-        This is just a placeholder for now.
+Rather than attempting to resolve this potentially unresolvable question, we
+propose a rubric &mdash; a scoring guide used to evaluate performance, a
+product, or a project &mdash; that teaches how to evaluate a given DID Method
+according to one's own requirements.
+      </p>
+      <p>
+This rubric presents a set of criteria which an <a>Evaluator</a> can apply to
+any DID Method based on the use cases most relevant to them. We avoid reducing
+the Evaluation to a single number because the criteria tend to be multidimensional
+and many of the possible responses are not necessarily good or bad. It is up
+to the <a>Evaluator</a> to understand how each response in each criteria might
+illuminate favorable or unfavorable consequences for their needs.
+      </p>
+      <p>
+While this rubric allows the evaluation of many aspects of decentralization of
+a DID Method, it is not exhaustive, and does not cover other factors that may
+affect selection or adoption of a particular Method, such as privacy, security,
+reliability, or efficiency.
       </p>
```

Comment:
<p>
The first draft of this document was based on _The Rubric for the Decentralization of DID Methods_, by Andrieu, et. al, published online at https://github.com/WebOfTrustInfo/rwot9-prague/blob/master/final-documents/decentralization-rubric.pdf
</p>